tests/unit/AGMDate.spec.ts

```diff
@@ -13,21 +16,23 @@ Vue.use(Vuetify)
 Vue.use(Vuelidate)
 
 const vuetify = new Vuetify({})
-const store = getVuexStore() as any // remove typings for unit tests
+setActivePinia(createPinia())
+const businessStore = useBusinessStore()
+const rootStore = useRootStore()
 
 describe('AgmDate', () => {
   let wrapper: Wrapper<AgmDate>
   let vm: any
 
   beforeEach(() => {
     // init store
-    store.state.ARFilingYear = 2019
-    store.state.arMinDate = '2019-01-01'
-    store.state.arMaxDate = '2019-12-31'
-    store.commit('setLegalType', 'CP')
-    store.commit('setLastAnnualReportDate', '2018-07-15')
+    rootStore.ARFilingYear = 2019
+    rootStore.arMinDate = '2019-01-01'
+    rootStore.arMaxDate = '2019-12-31'
```

Comment:
FUTURE: if there are setters (actions) for these, we should use them instead.

Reply:
Oh man. I should've done it that way while I was doing the work. I think I'll do this in the other ticket since it makes sense to use the setters instead of changing the state directly.

Reply:
It might not be worth fixing all the existing code, but for new code, yes!

(Note that some state doesn't have setters.)